blockchain/blockchain_test.go

```diff
@@ -1644,3 +1659,400 @@ func TestBlockchain_WriteFullBlock(t *testing.T) {
 	require.NotNil(t, db[hex.EncodeToHex(getKey(storage.CANONICAL, common.EncodeUint64ToBytes(header.Number)))])
 	require.NotNil(t, db[hex.EncodeToHex(getKey(storage.RECEIPTS, header.Hash.Bytes()))])
 }
+
+func TestDiskUsageWriteBatchAndUpdate(t *testing.T) {
+	const (
+		checkInterval  = 10000 //milliseconds
+		numberOfBlocks = 100
+		blockTime      = 2000 //milliseconds
+	)
+
+	jsonFile, err := os.Open("testblock.json")
+	require.NoError(t, err)
+
+	byteValue, _ := io.ReadAll(jsonFile)
+
+	receiptsJSONFile, err := os.Open("receipts.json")
+	require.NoError(t, err)
+
+	receiptsByteVaulue, _ := io.ReadAll(receiptsJSONFile)
+
+	blockWriter(t, checkInterval, blockTime, numberOfBlocks, byteValue, receiptsByteVaulue)
+}
+
+func TestDiskUsageWriteBatchAndUpdateNoTimeout(t *testing.T) {
```

Comment:
We don't need this test. One of them is enough (`TestDiskUsageWriteBatchAndUpdate`).